### PR TITLE
[issue] #103 The percentage difference is rounded and passes the tests

### PR DESCRIPTION
**Pull request by jesusfj710**
https://github.com/wswebcreation/webdriver-image-comparison/issues/103

### CODE REVIEW

File: lib/methods/images.ts [hunk 1]
```diff
@@ -246,7 +246,8 @@ export async function executeImageCompare(
 
   // 5.		Execute the compare and retrieve the data
   const data: CompareData = await compareImages(readFileSync(baselineFilePath), readFileSync(actualFilePath), compareOptions);
-  const misMatchPercentage = imageCompareOptions.rawMisMatchPercentage
+  const misMatchPercentage = data.rawMisMatchPercentage;
+  const reportMisMatchPercentage = imageCompareOptions.rawMisMatchPercentage
     ? data.rawMisMatchPercentage
```
Comment (wswebcreation) [inline lib/methods/images.ts:251]:
```suggestion
    ? rawMisMatchPercentage
```

File: lib/methods/images.ts [hunk 2]
```diff
@@ -246,7 +246,8 @@ export async function executeImageCompare(
 
   // 5.		Execute the compare and retrieve the data
   const data: CompareData = await compareImages(readFileSync(baselineFilePath), readFileSync(actualFilePath), compareOptions);
-  const misMatchPercentage = imageCompareOptions.rawMisMatchPercentage
+  const misMatchPercentage = data.rawMisMatchPercentage;
```
Comment (wswebcreation) [inline lib/methods/images.ts:249]: When we're touching this code, can we make it more clear by changing the name of the const? We then also need to change it on multiple places, including line 254/255

```suggestion
  const rawMisMatchPercentage = data.rawMisMatchPercentage;
```

File: lib/methods/images.ts [hunk 3]
```diff
@@ -246,7 +246,8 @@ export async function executeImageCompare(
...
+  const reportMisMatchPercentage = imageCompareOptions.rawMisMatchPercentage
     ? data.rawMisMatchPercentage
     : Number(data.rawMisMatchPercentage.toFixed(2));
```
Comment (wswebcreation) [inline lib/methods/images.ts:252]:
```suggestion
    : Number(rawMisMatchPercentage.toFixed(2));
```

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @jesusfj710

Thanks for this PR, I see it's in draft, but already added some minor comments. Are there any doubts/concerns that you have regarding this PR?

**Comment by jesus-e2y** [comment]
I'm sorry I haven't had time to check it, but I need to find the right time. I will probably go over and do another draft that is more complete and then we can discuss it.

**Comment by jesusfj710** [comment]
I'm sorry I haven't had time to check it, but I need to find the right time. I will probably go over and do another draft that is more complete and then we can discuss it.

**Comment by jesusfj710** [comment]
@wswebcreation maybe now? 🤔

**Comment by wswebcreation** [comment]
Thanks @jesusfj710 

I'll check this one later. This might be a breaking change and will need to update at least the https://github.com/wswebcreation/wdio-image-comparison-service module.

Will put this on my TODO list.

**Comment by wswebcreation** [comment]
Thanks for you work. 
This will be a breaking change because it will now by default return `0.00` instead of `0`. I'm going to release this in a few days because I want to fix some more things

**Review by wswebcreation — APPROVED** [review]

**Comment by wswebcreation** [comment]
@jesusfj710 

I've been thinking about this and I had a lot of breaking tests, so I reverted your PR and updated it to the following https://github.com/wswebcreation/webdriver-image-comparison/releases/tag/v1.0.2. I hope you don't mind

**Comment by jesusfj710** [comment]
Ok, but at some point this should be handled better or at least made clear in the `readme.MD` or in the wiki how this can affect the use of the library.

**Pull request merged**